app.js

```diff
@@ -22,3 +86,132 @@ menulink.forEach((e) => {
+  const techArr = projectArr[i].technologies;
+  for (let j = 0; j < techArr.length; j += 1) {
+    uList += `
+                <li>${techArr[j]}</li>
+              `;
+  }
+  myCard += `<article class="work-card">
+          <figure class="work-img">
+            <img src="${myObj.imae}" alt="work screenshot" />
+          </figure>
+          <div class="work-text">
+            <header>
+              <h2>${myObj.name}</h2>
+            </header>
+            <div class="work-tools">
+            <ul>
+              ${uList}
+              </ul>
+            </div>
+            <div class="project-btn">
+              <button type="button" class='work-btn' ownIndex=${i}>See Project</button>
+            </div>
+          </div>
+        </article>`;
+}
+cardContainer.innerHTML = myCard;
+
+const showModal = () => {
+  const btn = document.querySelectorAll('.work-btn');
+  const modalContainer = document.createElement('section');
+  modalContainer.className = 'modal-container';
+  const modalMobile = document.createElement('section');
+  modalMobile.className = 'modal-container-mobile';
+  btn.forEach((e) => {
+    e.addEventListener('click', () => {
+      const index = e.getAttribute('ownIndex');
+      const forLaptop = modalElem(projectArr[index]);
+      const forMobile = modalMob(projectArr[index]);
+      const body = document.querySelector('body');
+      body.append(modalContainer);
+      body.append(modalMobile);
+      const modalContain = document.querySelector('.modal-container');
+      const mobileContain = document.querySelector('.modal-container-mobile');
+      mobileContain.innerHTML = forMobile;
+      modalContain.innerHTML = forLaptop;
+      const closeModal = document.querySelectorAll('.close-modal');
+      closeModal.forEach((e) => {
+        e.addEventListener('click', () => {
+          modalMobile.remove();
+          modalContain.remove();
+        });
+      });
+    });
+  });
+};
+showModal();
+
+const modalElem = (arr) => `
+ <div class="modal-body">
+        <figure class="close-img">
+          <img class="close-modal" src="./img/close-x.png" alt="" />
+        </figure>
+        <figure class="modal-img">
+          <img class="img-file" src="./img/card1.png" alt="" />
+        </figure>
+        <div class="modal-tittle">
+          <div class="heading">
+            <h2>${arr.name}</h2>
+          </div>
+          <div class="modal-btns">
+            <a href="" target="_blank"><button>See Live <img src="./img/see-live.png" alt=""></i></button></a>
+            <a href="" target="_blank"><button>See Source <img src="./img/gh.png" alt="github"></button></a>
+          </div>
+        </div>
+        <div class="technologies">
+          <ul>
+          ${uList}
+          </ul>
+        </div>
+        <div class="desc">
+          <p>
+            ${arr.description}
+          </p>
+        </div>
+      </div>
+`;
+
+const modalMob = (arr) => `
+  <div class="modal-body">
```

Comment:
- [ ] @team great work, however your popup in mobile view has an empty space on the top kindly remove it:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/61408860/188835549-8939f340-f814-46b5-877d-d2f7814dd966.png">